x11-terms/{st,st-terminfo}: add proxied maintainer

ferki opened the pull request:
This PR is an attempt to add myself as proxied maintainer of x11-terms/st and x11-terms/st-terminfo (as briefly discussed with @gyakovlev via IRC). There's also a minor indent fix on a separate commit - take it or leave it :)

Please review and merge, or let me know how to improve it further.

gyakovlev commented:
looks good, but let's not add proxy-maint team.
let's just leave 2 of us.
non gentoo.org maintainer can be there without proxy team with another developer without team if agreed upon.
and I'm part of proxy-maint team, so that's fine with me too.


also feel free to move yourself on top/as first, but that'll make you primary assignee for bugs (not that we get many)
it's ok if you don't do that.

ferki commented:
@gyakovlev: thanks for the feedback! I have rebased this PR on top of current default branch, removed the proxy-maint team, and moved my entry to the top.
